taipy/gui/_renderers/builder.py

```diff
@@ -427,7 +429,7 @@ def _get_lov_adapter(self, var_name: str, property_name: t.Optional[str] = None,
             if adapter is not None:
                 self.__gui._add_adapter_for_type(var_type, adapter)  # type: ignore
 
-            if len(lov) > 0:
+            if default_lov is not None and len(lov) > 0:
```

Comment:
can we say if default_lov is not None and lov: ?

Reply:
so that we don't have to call the `len` function

Reply:
We can 👍